reference/webhooks.v3.yml

```
@@ -1,463 +1,644 @@
+  contact: {}
+servers:
+  - url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v3'
+    variables:
+      $$.env.store_hash:
+        default: '{$$.env.store_hash}'
+  - url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2'
+    variables:
+      $$.env.store_hash:
+        default: '{$$.env.store_hash}'
 paths:
   /hooks:
-    post:
-      responses:
-        '200':
-          $ref: '#/components/responses/webhook_Resp'
-        '422':
-          $ref: '#/components/responses/422_UnprocessableEntity'
-      summary: Create a Webhook
-      description: Creates a webhook. Only one webhook at a time can be created. Custom headers can be added. Destination URL must be served on port 443 (custom ports are not currently supported).
-      operationId: createWebhooks
+    get:
+      operationId: getAllWebhooks
+      summary: Get Webhooks
+      description: |-
+        Returns a list of all webhooks on a store associated to the `client_id` used to authenticate the request.
+
+        <div class="HubBlock--callout">
+        <div class="CalloutBlock--info">
+        <div class="HubBlock-content">
+
+
+        > ### Note
+        > * Webhooks endpoints are available on both the V2 and V3 REST API. 
+        > * BigCommerce determines the `client_id` from the `access_token`.
+
+        </div>
+        </div>
+        </div>
       parameters:
         - $ref: '#/components/parameters/Accept'
         - $ref: '#/components/parameters/Content-Type'
+      responses:
+        '200':
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Webhook'
+                  meta:
+                    type: object
+                    properties:
+                      pagination:
+                        $ref: '#/components/schemas/Pagination'
+              examples:
+                Example:
+                  value:
+                    data:
+                      - id: 18048287
+                        client_id: m9r6keqmo7h7f23btnpwernbez1kglkl
+                        store_hash: sftg45fsd
+                        created_at: 1561488106
+                        updated_at: 1561488106
+                        scope: '`store/order/*`'
+                        destination: 'https://665b65a6.ngrok.io/webhooks`'
+                        is_active: true
+                        headers:
+                          custom: string
+                    meta:
+                      pagination:
+                        offset: 0
+                        limit: 0
+                        total_items: 0

```

V3 pagination which is used for webhooks as well has different structure: 
```suggestion
                      pagination:
                        total: 0
                        count: 0
                        per_page: 0
                        current_page: 0
                        total_pages: 0
                        links:
                          previous: ?limit=0&page=0
                          current: ?limit=0&page=0
                          next: ?limit=0&page=0

```